Create padding utility classes for print

belanglos opened the pull request:
# tl;dr

We would love to use the class `p-print-3` to apply padding to an element only when printing, to have a nice looking pdf or printout. Plz merge! 😍 

# Change

Bootstrap has some built-in utility classes to e.g. only display elements if the user is printing a website (i.e. within the @media print media query. Print utility classes for additional properties have to explicitly be enabled according to: https://getbootstrap.com/docs/5.3/utilities/api/#print

This change is _modifying_ the utility classes for **padding** (according to https://getbootstrap.com/docs/5.3/utilities/api/#modify-utilities), and thus additionally creating utility classes p-print-0 to p-print-5, while leaving all other classes untouched.


The created classes:

```css
.p-print-0 {
	padding: 0 !important;
}

.p-print-1 {
	padding: 0.325rem !important;
}

.p-print-2 {
	padding: 0.65rem !important;
}

.p-print-3 {
	padding: 1.3rem !important;
}

.p-print-4 {
	padding: 1.95rem !important;
}

.p-print-5 {
	padding: 3.9rem !important;
}

```

# Our use case

We want to create beautiful printouts (or save to pdfs) with nice padding.

Currently:
![pdf-bad](https://github.com/KTH/kth-style/assets/3601810/11cf23c9-20a5-44e4-a1b3-ae16dcde45ba)

How things could be:
![pdf-good](https://github.com/KTH/kth-style/assets/3601810/ae9417d0-37ef-4e0d-9e15-32a983d9689e)

saraervik commented:
kth-style-packages will soon be deprecated, therefor closing this PR

kstarzecki commented:
Hey,

Any progress?

Thanks,
Chris

Sent from my iPhone

> On 28 Aug 2019, at 09:16, Sara Ervik <notifications@github.com> wrote:
> 
> ﻿
> https://github.com/KTH/kth-style-packages/network/alerts
> 
> Update packages to fix Github's security alert
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/KTH/kth-style-packages/pull/17
> 
> Commit Summary
> 
> Update dependencies due to security vulnerabilities
> File Changes
> 
> M packages/kth-style-web/package.json (6)
> Patch Links:
> 
> https://github.com/KTH/kth-style-packages/pull/17.patch
> https://github.com/KTH/kth-style-packages/pull/17.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

belanglos committed:
feat: create padding utility classes for print

Bootstrap has some built-in utility classes to e.g. only display elements
if the user is printing a website (i.e. within the @media print media query.
Print utility classes for additional properties have to explicitly be enabled.

This change is creating utility classes p-print-0 to p-print-5.
